src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: 'Roboto font', sans-serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.header {
+  height: 60px;
+}
+
+.nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-transform: uppercase;
+  margin: 0 50px;
+}
+
+.nav__logo {
+  margin-top: 10px;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  list-style-type: none;
+}
+
+.nav__item {
+  padding-right: 20px;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: var(--nav-main-color);
+}
+
+.nav__link:hover {
+  transition: 0.3s;
+  color: var(--active-link-color);
+}
+
+.is-active::after {
+  color: var(--active-link-color);
+  display: block;
+  content: '';
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+  background: var(--active-link-color);
+  position: absolute;
+  bottom: 0;
+}
```

Comment:
The `.is-active::after` pseudo-element uses `position: absolute;` without a relative positioned parent. Ensure that the parent element of `.is-active` is set to `position: relative;` to avoid unexpected positioning issues.